samples/attested_tls/non_enc_client/verify_callback.cpp

```diff
@@ -22,13 +23,31 @@
 #include "../common/tls_server_enc_mrenclave.h"
 #include "../common/tls_server_enc_pubkey.h"
 
-bool verify_mrsigner(
+/**
+ * Helper function used to make the claim-finding process more convenient. Given
+ * the claim name, claim list, and its size, returns the claim with that claim
+ * name in the list.
+ */
+const oe_claim_t* find_claim(
+    const oe_claim_t* claims,
+    size_t claims_size,
+    const char* name)
+{
+    for (size_t i = 0; i < claims_size; i++)
+    {
+        if (strcmp(claims[i].name, name) == 0)
+            return &(claims[i]);
+    }
+    return NULL;
```

Comment:
```suggestion
    return nullptr;
```

Reply:
`-std=c++11` flag was not set in non-enc Makefile so `nullptr` here raised the "error: ‘nullptr’ was not declared in this scope" for ACC1604 gcc Debug/Release LVI. So I kept it as `NULL` here.

Reply:
Should look into why that failed and fix it.  Ok to do so in a separate PR.

Reply:
> Should look into why that failed and fix it. Ok to do so in a separate PR.

I added -std=c++11 flag and now `nullptr` passed the bors try.